### ISSUE TITLE
Output base 64

### ISSUE THREAD

**Issue by joshuawprice** [FiatPrincipiaFalsa/Encryption#3]
Rather than outputting binary represented using text, why don't we output base 64, represented as text?

This'll make the output much smaller and in my opinion also make it look much better.

**Comment by FiatPrincipiaFalsa** [comment]
i agree this will make the output much more manageable for the user and thus it will be more friendly, i am open to implementing this idea in the near future

**Comment by FiatPrincipiaFalsa** [comment]
I'm free for i large period of time, how would you suggest i go about implementing the base 64 output?

**Comment by joshuawprice** [comment]
I would create a function that can take in the binary strings as you have them now, and prints the base64. Then you can just change the current print functions to the new method and you'll be done.

As for creating this new method, I would have it convert your binary strings to the python "bytes" data type, and then use [this](https://docs.python.org/3/library/base64.html) module to do the conversion to base64 and print out the result.
